### PR TITLE
Add more specificity to CSS rule and additional classes to menu

### PR DESCRIPTION
**Pull request by jnatalzia**
### This PR will...
Fix up additional style bleeding issues with the right click menu (version text specifically)

### Why is this Pull Request needed?
We control our styles

### Are there any points in the code the reviewer needs to double check?
Nah

### Are there any Pull Requests open in other repos which need to be merged with this?
Nah

#### Addresses Issue(s):

JW8-10708 

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

File: src/css/jwplayer/imports/reset.less [hunk 1]
```diff
@@ -1,17 +1,18 @@
-.jw-reset {
+.jwplayer .jw-reset {
```
Comment (zetagame) [inline src/css/jwplayer/imports/reset.less]: Side effect: Shelf Widget style bleeding

Comment (jnatalzia) [reply src/css/jwplayer/imports/reset.less]: Great catch

Comment (robwalch) [reply src/css/jwplayer/imports/reset.less]: yikes!

File: src/css/jwplayer/imports/reset.less [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-.jw-reset-text {
+.jwplayer .jw-reset-text {
     color: inherit;
     background-color: transparent;
     padding: 0;
     margin: 0;
     float: none;
     font-family: Arial, Helvetica, sans-serif;
     font-size: 1em;
+    font-weight: normal;
```
Comment (robwalch) [inline src/css/jwplayer/imports/reset.less]: All we need is this on right click text

**Pull request merged**